browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -769,8 +769,9 @@ void RewardsDOMHandler::OnNotificationDeleted(
     brave_rewards::RewardsNotificationService* rewards_notification_service,
     const brave_rewards::RewardsNotificationService::RewardsNotification&
         notification) {
-  if (notification.type_ == brave_rewards::RewardsNotificationService::RewardsNotificationType::REWARDS_NOTIFICATION_GRANT &&
-    web_ui()->CanCallJavascript()) {
+  if (notification.type_ ==
```

Comment:
why is this needed?

Reply:
@SergeyZhukovsky do you remember why?

Reply:
IIRC to remove a grant message from a settings page when we claimed it via panel and the page is active at the same moment.

Reply:
that should be done via observers. So probably one observer is not handled correctly on settings page. We need to remove this as it could trigger multiple success messages

Reply:
Temporary excluded with pre-processor for Android only.
Opened refactoring issue: https://github.com/brave/brave-browser/issues/6316